diff -ruN a/Makefile b/Makefile
--- a/Makefile	2023-01-10 11:11:11.000000000 +0000
+++ b/Makefile	2023-04-13 12:58:34.537146561 +0100
@@ -2,7 +2,7 @@
 #
 # Useful targets are: release, install, uninstall.
 
-default: build/debug/mujs build/debug/mujs-pp
+default: build/release/mujs build/release/mujs-pp
 
 CFLAGS = -std=c99 -pedantic -Wall -Wextra -Wno-unused-parameter
 
@@ -19,6 +19,10 @@
   VERSION = $(patsubst mujs-%,%,$(notdir $(CURDIR)))
 endif
 
+ifeq "$(host)" "mingw"
+  EXT := .exe
+endif
+
 ifeq ($(shell uname),Darwin)
   SO = dylib
 else
@@ -99,7 +103,7 @@
 build/release/libmujs.a: build/release/libmujs.o
 	$(AR) cr $@ $^
 build/release/mujs: main.c build/release/libmujs.o
-	$(CC) $(CFLAGS) $(OPTIM) -o $@ $^ -lm -DHAVE_READLINE -lreadline
+	$(CC) $(CFLAGS) $(OPTIM) -o $@ $^ -lm
 build/release/mujs-pp: pp.c build/release/libmujs.o
 	$(CC) $(CFLAGS) $(OPTIM) -o $@ $^ -lm
 
@@ -119,8 +123,8 @@
 	install -d $(DESTDIR)$(bindir)
 	install -m 644 mujs.h $(DESTDIR)$(incdir)
 	install -m 644 build/release/mujs.pc $(DESTDIR)$(libdir)/pkgconfig
-	install -m 755 build/release/mujs $(DESTDIR)$(bindir)
-	install -m 755 build/release/mujs-pp $(DESTDIR)$(bindir)
+	install -m 755 build/release/mujs$(EXT) $(DESTDIR)$(bindir)
+	install -m 755 build/release/mujs-pp$(EXT) $(DESTDIR)$(bindir)
 
 install-static: install-common build/release/libmujs.a
 	install -m 644 build/release/libmujs.a $(DESTDIR)$(libdir)
