diff -ruN a/Makefile b/Makefile
--- a/Makefile	2025-06-23 13:15:36.000000000 +0100
+++ b/Makefile	2025-06-24 16:24:03.199185593 +0100
@@ -2,7 +2,7 @@
 #
 # Useful targets are: release, install, uninstall.
 
-default: build/debug/mujs build/debug/mujs-pp
+default: build/release/mujs build/release/mujs-pp
 
 CFLAGS = -std=c99 -pedantic -Wall -Wextra -Wno-unused-parameter
 
@@ -19,6 +19,10 @@
   VERSION = $(patsubst mujs-%,%,$(notdir $(CURDIR)))
 endif
 
+ifeq "$(host)" "mingw"
+  EXT := .exe
+endif
+
 ifeq ($(shell uname),Darwin)
   SO = dylib
 else
@@ -126,8 +130,8 @@
 	install -d $(DESTDIR)$(bindir)
 	install -m 644 mujs.h $(DESTDIR)$(incdir)
 	install -m 644 build/release/mujs.pc $(DESTDIR)$(libdir)/pkgconfig
-	install -m 755 build/release/mujs $(DESTDIR)$(bindir)
-	install -m 755 build/release/mujs-pp $(DESTDIR)$(bindir)
+	install -m 755 build/release/mujs$(EXT) $(DESTDIR)$(bindir)
+	install -m 755 build/release/mujs-pp$(EXT) $(DESTDIR)$(bindir)
 
 install-static: install-common build/release/libmujs.a
 	install -m 644 build/release/libmujs.a $(DESTDIR)$(libdir)
