From 491c6a77d31bdd106d42f2ba8ddf9aaf88d3cad7 Mon Sep 17 00:00:00 2001
From: Ray Griffin <raymond.griffin@protonmail.com>
Date: Tue, 1 Mar 2022 07:32:16 +0000
Subject: [PATCH] fix map downloads

---
 code/client/cl_main.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/code/client/cl_main.c b/code/client/cl_main.c
index 038078f6..306d9c73 100644
--- a/code/client/cl_main.c
+++ b/code/client/cl_main.c
@@ -947,7 +947,7 @@ static void CL_CompleteDemoName( char *args, int argNum )
 		Field_CompleteFilename( "demos", ".urtdemo", qfalse, FS_MATCH_ANY | FS_MATCH_STICK );
 #else
 		Field_CompleteFilename( "demos", ".dm_??", qfalse, FS_MATCH_ANY | FS_MATCH_STICK );
-#endif	
+#endif
 		FS_SetFilenameCallback( NULL );
 	}
 }
@@ -4121,7 +4121,8 @@ void CL_Init( void ) {
 
 	rconAddress = Cvar_Get ("rconAddress", "", 0);
 
-	cl_allowDownload = Cvar_Get( "cl_allowDownload", "1", CVAR_ARCHIVE_ND );
+	//cl_allowDownload = Cvar_Get( "cl_allowDownload", "1", CVAR_ARCHIVE_ND );
+	cl_allowDownload = Cvar_Get( "cl_allowDownload", "1", CVAR_ROM );
 #ifdef USE_CURL
 	cl_mapAutoDownload = Cvar_Get( "cl_mapAutoDownload", "0", CVAR_ARCHIVE_ND );
 #ifdef USE_CURL_DLOPEN
-- 
2.35.1

