From d2abb6b7f3f123cfc4820f315ce6222df8bb3271 Mon Sep 17 00:00:00 2001
From: Ray Griffin <raymond.griffin@protonmail.com>
Date: Tue, 22 Feb 2022 16:53:26 +0000
Subject: [PATCH 1/2] Makefile Options

---
 Makefile | 36 +++++++++++++++---------------------
 1 file changed, 15 insertions(+), 21 deletions(-)

diff --git a/Makefile b/Makefile
index 1a2273bb..936aeeb1 100644
--- a/Makefile
+++ b/Makefile
@@ -23,27 +23,21 @@ ifeq ($(COMPILE_PLATFORM),mingw32)
   endif
 endif
 
-BUILD_CLIENT     = 1
-BUILD_SERVER     = 1
-
-USE_SDL          = 1
-USE_CURL         = 1
-USE_LOCAL_HEADERS= 0
-USE_VULKAN       = 1
-USE_OPENGL       = 1
-USE_OPENGL2      = 0
-USE_SYSTEM_JPEG  = 0
-USE_VULKAN_API   = 1
-
-USE_RENDERER_DLOPEN = 1
-
-CNAME            = quake3e
-DNAME            = quake3e.ded
-
-RENDERER_PREFIX  = $(CNAME)
-
-# valid options: opengl, vulkan, opengl2
-RENDERER_DEFAULT = opengl
+BUILD_CLIENT		= 1
+BUILD_SERVER		= 1
+USE_SDL				= 0
+USE_CURL			= 1
+USE_LOCAL_HEADERS	= 1
+USE_VULKAN			= 1
+USE_OPENGL			= 0
+USE_OPENGL2			= 0
+USE_SYSTEM_JPEG		= 0
+USE_VULKAN_API		= 1
+USE_RENDERER_DLOPEN	= 0
+CNAME				= quake3e
+DNAME				= quake3e.ded
+RENDERER_PREFIX		= $(CNAME)
+RENDERER_DEFAULT	= vulkan
 
 
 ifeq ($(V),1)
-- 
2.37.1

