From 71924f6f26be88f6b437f4c785d23d22bfe78dfd Mon Sep 17 00:00:00 2001
From: Ray Griffin <raymond.griffin@protonmail.com>
Date: Wed, 23 Feb 2022 19:43:02 +0000
Subject: [PATCH] Set game version

---
 code/qcommon/q_shared.h | 43 +++++++++++++++++++----------------------
 1 file changed, 20 insertions(+), 23 deletions(-)

diff --git a/code/qcommon/q_shared.h b/code/qcommon/q_shared.h
index 954ecc2d..301829b6 100644
--- a/code/qcommon/q_shared.h
+++ b/code/qcommon/q_shared.h
@@ -26,31 +26,28 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 // q_shared.h -- included first by ALL program modules.
 // A user mod should never modify this file
 
-#define Q3_VERSION            "UrtSlim-Q3e"
+#define Q3_VERSION              "ioQ3 1.35 urt 4.3.4"
 #ifndef SVN_VERSION
   #define SVN_VERSION Q3_VERSION
 #endif
-#define CLIENT_WINDOW_TITLE   "Urban Terror Slim"
-#define CONSOLE_WINDOW_TITLE  "Urt Console"
-// 1.32 released 7-10-2002
+#define CLIENT_WINDOW_TITLE     "Quake3e: Urban Terror"
+#define CONSOLE_WINDOW_TITLE    "Quake3e: Urt Console"
 
-//#define DEFAULT_GAME			"edawn"
-#define DEFAULT_GAME				"q3ut4"
-
-#define BASEGAME				"q3ut4"
-#define BASEDEMO				"demoq3"
-#define BASETA					"missionpack"
-#define STEAMPATH_NAME			"Quake 3 Arena"
-#define STEAMPATH_APPID			"2200"
+#define DEFAULT_GAME            "q3ut4"
+#define BASEGAME                "q3ut4"
+#define BASEDEMO                "demoq3"
+#define BASETA                  "missionpack"
+#define STEAMPATH_NAME          "Quake 3 Arena"
+#define STEAMPATH_APPID         "2200"
 
 #define MAX_TEAMNAME            32
-#define MAX_MASTER_SERVERS      5	// number of supported master servers
+#define MAX_MASTER_SERVERS      5
 
-#define GAMENAME_FOR_MASTER		"Quake3Arena"
-#define HEARTBEAT_FOR_MASTER	"QuakeArena-1"
+#define GAMENAME_FOR_MASTER     "Quake3Arena"
+#define HEARTBEAT_FOR_MASTER    "QuakeArena-1"
 
-#define DEMOEXT	"dm_"			// standard demo extension
-#define URTDEMOEXT "urtdemo"
+#define DEMOEXT                 "dm_"
+#define URTDEMOEXT              "urtdemo"
 
 #ifdef _MSC_VER
 
@@ -247,7 +244,7 @@ typedef int		clipHandle_t;
 #define	MAX_INFO_VALUE		1024
 
 #define MAX_USERINFO_LENGTH (MAX_INFO_STRING-13) // incl. length of 'connect ""' or 'userinfo ""' and reserving one byte to avoid q3msgboom
-													
+
 #define	BIG_INFO_STRING		8192  // used for system info key only
 #define	BIG_INFO_KEY		  8192
 #define	BIG_INFO_VALUE		8192
@@ -557,7 +554,7 @@ void AddPointToBounds( const vec3_t v, vec3_t mins, vec3_t maxs );
 static ID_INLINE int VectorCompare( const vec3_t v1, const vec3_t v2 ) {
 	if (v1[0] != v2[0] || v1[1] != v2[1] || v1[2] != v2[2]) {
 		return 0;
-	}			
+	}
 	return 1;
 }
 
@@ -1137,7 +1134,7 @@ typedef struct {
 #define	MAX_STATS				16
 #define	MAX_PERSISTANT			16
 #define	MAX_POWERUPS			16
-#define	MAX_WEAPONS				16		
+#define	MAX_WEAPONS				16
 
 #define	MAX_PS_EVENTS			2
 
@@ -1257,7 +1254,7 @@ typedef struct usercmd_s {
 	int				serverTime;
 	int				angles[3];
 	int 			buttons;
-	byte			weapon;           // weapon 
+	byte			weapon;           // weapon
 	signed char	forwardmove, rightmove, upmove;
 } usercmd_t;
 
@@ -1337,7 +1334,7 @@ typedef struct entityState_s {
 typedef enum {
 	CA_UNINITIALIZED,
 	CA_DISCONNECTED, 	// not talking to a server
-	CA_AUTHORIZING,		// not used any more, was checking cd key 
+	CA_AUTHORIZING,		// not used any more, was checking cd key
 	CA_CONNECTING,		// sending request packets to the server
 	CA_CHALLENGING,		// sending challenge packets to the server
 	CA_CONNECTED,		// netchan_t established, getting gamestate
@@ -1347,7 +1344,7 @@ typedef enum {
 	CA_CINEMATIC		// playing a cinematic or a static pic, not connected to a server
 } connstate_t;
 
-// font support 
+// font support
 
 #define GLYPH_START 0
 #define GLYPH_END 255
-- 
2.35.1

