From 41992110dc8c8e1e33c631efa4f2441b5c6a5100 Mon Sep 17 00:00:00 2001
From: Ray Griffin <raymond.griffin@protonmail.com>
Date: Sun, 19 Dec 2021 14:33:29 +0000
Subject: [PATCH] makefile build opts

---
 Makefile | 37 +++++++++++++++----------------------
 1 file changed, 15 insertions(+), 22 deletions(-)

diff --git a/Makefile b/Makefile
index e4e4d2d3..0a53f8bd 100644
--- a/Makefile
+++ b/Makefile
@@ -23,28 +23,21 @@ ifeq ($(COMPILE_PLATFORM),mingw32)
   endif
 endif

-BUILD_CLIENT     = 1
-BUILD_SERVER     = 1
-
-USE_SDL          = 1
-USE_CURL         = 1
-USE_LOCAL_HEADERS= 0
-USE_VULKAN       = 1
-USE_OPENGL       = 1
-USE_OPENGL2      = 0
-USE_SYSTEM_JPEG  = 0
-USE_VULKAN_API   = 1
-
-USE_RENDERER_DLOPEN = 1
-
-CNAME            = quake3e
-DNAME            = quake3e.ded
-
-RENDERER_PREFIX  = $(CNAME)
-
-# valid options: opengl, vulkan, opengl2
-RENDERER_DEFAULT = opengl
-
+BUILD_CLIENT        = 1
+BUILD_SERVER        = 1
+USE_CURL            = 1
+USE_LOCAL_HEADERS   = 0
+USE_OPENGL          = 0
+USE_OPENGL2         = 0
+USE_RENDERER_DLOPEN = 0
+USE_SDL             = 0
+USE_SYSTEM_JPEG     = 0
+USE_VULKAN          = 1
+USE_VULKAN_API      = 1
+RENDERER_DEFAULT    = vulkan
+CNAME               = quake3e
+DNAME               = quake3e.ded
+RENDERER_PREFIX     = $(CNAME)

 ifeq ($(V),1)
 echo_cmd=@:
--
2.34.1

