From 081858c980e3be585746343ae4c334c0a4d249ea Mon Sep 17 00:00:00 2001
From: rorgoroth <77244135+rorgoroth@users.noreply.github.com>
Date: Mon, 23 Sep 2024 17:59:08 +0100
Subject: [PATCH] src/windows/meson.build: disable khr-headers wrap

---
 src/windows/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/windows/meson.build b/src/windows/meson.build
index 659cd5c..bedb05e 100644
--- a/src/windows/meson.build
+++ b/src/windows/meson.build
@@ -26,7 +26,7 @@ game_src += windows.compile_resources('res/game.rc',
   include_directories: rc_inc,
 )
 
-client_deps += subproject('khr-headers').get_variable('khr_headers_dep')
+#client_deps += subproject('khr-headers').get_variable('khr_headers_dep')
 
 if get_option('software-sound').allowed()
   client_src += files('wave.c', 'dsound.c')
-- 
2.46.1

