From 9c70dcf4e43972f305934f4f6b7db972538398fa Mon Sep 17 00:00:00 2001
From: Ray Griffin <raymond.griffin@protonmail.com>
Date: Sun, 4 Sep 2022 20:06:14 +0100
Subject: [PATCH] Makefile.local diff

---
 Makefile.local | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 Makefile.local

diff --git a/Makefile.local b/Makefile.local
new file mode 100644
index 00000000..789d3dd5
--- /dev/null
+++ b/Makefile.local
@@ -0,0 +1,14 @@
+PLATFORM            = mingw64
+ARCH                = x86_64
+
+BUILD_CLIENT        = 1
+BUILD_SERVER        = 1
+USE_SDL             = 0
+USE_CURL            = 1
+USE_LOCAL_HEADERS   = 0
+USE_VULKAN          = 1
+USE_OPENGL          = 0
+USE_OPENGL2         = 0
+USE_SYSTEM_JPEG     = 0
+USE_VULKAN_API      = 1
+USE_RENDERER_DLOPEN = 0
-- 
2.37.1

