From 1aea152d8c71299dce743838cedcca89002246af Mon Sep 17 00:00:00 2001
From: Ray Griffin <raymond.griffin@protonmail.com>
Date: Tue, 6 Sep 2022 15:16:54 +0100
Subject: [PATCH] Local Makefile

---
 Makefile.local | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)
 create mode 100644 Makefile.local

diff --git a/Makefile.local b/Makefile.local
new file mode 100644
index 0000000..19716ec
--- /dev/null
+++ b/Makefile.local
@@ -0,0 +1,17 @@
+CC=x86_64-w64-mingw32-gcc
+PLATFORM=mingw64
+ARCH=x86_64
+
+BUILD_CLIENT=1
+BUILD_SERVER=1
+
+USE_SDL=0
+USE_CURL=1
+USE_LOCAL_HEADERS=0
+USE_VULKAN=1
+USE_OPENGL=0
+USE_OPENGL2=0
+USE_SYSTEM_JPEG=0
+USE_VULKAN_API=1
+USE_RENDERER_DLOPEN=0
+RENDERER_DEFAULT=vulkan
-- 
2.37.1

