diff -ruN a/loader/CMakeLists.txt b/loader/CMakeLists.txt
--- a/loader/CMakeLists.txt	2022-07-17 06:28:13.038638200 +0100
+++ b/loader/CMakeLists.txt	2022-07-17 06:27:35.374212000 +0100
@@ -323,8 +323,6 @@
 endif()
 
 # Generate pkg-config file.
-include(FindPkgConfig QUIET)
-if(PKG_CONFIG_FOUND)
     set(VK_API_VERSION "${LOADER_GENERATED_HEADER_VERSION}")
     if(WIN32 AND NOT MINGW)
         set(VULKAN_LIB_SUFFIX "-1")
@@ -341,7 +339,6 @@
     set(PRIVATE_LIBS "${PRIVATE_LIBS} -lshlwapi -lcfgmgr32")
     configure_file("vulkan.pc.in" "vulkan.pc" @ONLY)
     install(FILES "${CMAKE_CURRENT_BINARY_DIR}/vulkan.pc" DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
-endif()
 
 target_link_libraries(vulkan PRIVATE Vulkan::Headers)
 add_library(Vulkan::Vulkan ALIAS vulkan)
