diff -ruN a/loader/CMakeLists.txt b/loader/CMakeLists.txt
--- a/loader/CMakeLists.txt	2024-04-19 17:48:08.000000000 +0100
+++ b/loader/CMakeLists.txt	2024-04-20 07:46:00.947325106 +0100
@@ -22,8 +22,9 @@
 add_library(loader_specific_options INTERFACE)
 target_link_libraries(loader_specific_options INTERFACE loader_common_options Vulkan::Headers)
 target_include_directories(loader_specific_options INTERFACE ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/generated ${CMAKE_CURRENT_BINARY_DIR})
+configure_file("vulkan_own.pc.in" "vulkan_own.pc" @ONLY)
 
-if(WIN32)
+if(WIN32 AND NOT MINGW)
 
     if(ENABLE_WIN10_ONECORE)
         # Note: When linking your app or driver to OneCore.lib, be sure to remove any links to non-umbrella libs (such as
@@ -334,7 +335,7 @@
     endif()
 endif()
 
-if(WIN32)
+if(MSVC)
     add_library(loader-opt STATIC ${OPT_LOADER_SRCS})
     target_link_libraries(loader-opt PUBLIC loader_specific_options)
     add_dependencies(loader-opt loader_asm_gen_files)
@@ -380,21 +381,8 @@
     add_dependencies(vulkan loader_asm_gen_files)
 
 else()
-    if(APPLE)
-        option(APPLE_STATIC_LOADER "Build a loader that can be statically linked. Intended for Chromium usage/testing.")
-        mark_as_advanced(APPLE_STATIC_LOADER)
-    endif()
-
-    if(APPLE_STATIC_LOADER)
-        add_library(vulkan STATIC)
-        target_compile_definitions(vulkan PRIVATE APPLE_STATIC_LOADER)
-
-        message(WARNING "The APPLE_STATIC_LOADER option has been set. Note that this will only work on MacOS and is not supported "
-                "or tested as part of the loader. Use it at your own risk.")
-    else()
-        add_library(vulkan SHARED)
-    endif()
-
+    add_library(vulkan STATIC)
+    target_compile_definitions(vulkan PRIVATE BUILD_STATIC_LOADER)
     target_sources(vulkan PRIVATE ${NORMAL_LOADER_SRCS} ${OPT_LOADER_SRCS})
 
     add_dependencies(vulkan loader_asm_gen_files)
@@ -488,7 +476,7 @@
 target_link_libraries(vulkan PRIVATE Vulkan::Headers)
 add_library(Vulkan::Loader ALIAS vulkan)
 
-if (APPLE_STATIC_LOADER)
+if (BUILD_STATIC_LOADER)
     # TLDR: This feature only exists at the request of Google for Chromium. No other project should use this!
     message(NOTICE "Apple STATIC lib: it will be built but not installed, and vulkan.pc and VulkanLoaderConfig.cmake won't be generated!")
     return()
diff -ruN a/loader/loader.h b/loader/loader.h
--- a/loader/loader.h	2024-04-19 17:48:08.000000000 +0100
+++ b/loader/loader.h	2024-04-20 07:46:00.947325106 +0100
@@ -33,7 +33,9 @@
 #include "cJSON.h"
 
 // Declare the once_init variable
+#if defined(_WIN32) && !defined(LOADER_DYNAMIC_LIB)
 LOADER_PLATFORM_THREAD_ONCE_EXTERN_DEFINITION(once_init)
+#endif
 
 static inline VkPhysicalDevice loader_unwrap_physical_device(VkPhysicalDevice physicalDevice) {
     struct loader_physical_device_tramp *phys_dev = (struct loader_physical_device_tramp *)physicalDevice;
diff -ruN a/loader/loader.rc.in b/loader/loader.rc.in
--- a/loader/loader.rc.in	2024-04-19 17:48:08.000000000 +0100
+++ b/loader/loader.rc.in	2024-04-20 07:46:00.947325106 +0100
@@ -19,7 +19,11 @@
 // Author: Charles Giessen <charles@lunarg.com>
 //
 
+#ifdef __MINGW64__
+#include <winresrc.h>
+#else // MSVC
 #include "winres.h"
+#endif
 
 // All set through CMake
 #define VER_FILE_VERSION ${LOADER_VER_FILE_VERSION}
diff -ruN a/loader/loader_windows.c b/loader/loader_windows.c
--- a/loader/loader_windows.c	2024-04-19 17:48:08.000000000 +0100
+++ b/loader/loader_windows.c	2024-04-20 07:46:00.947325106 +0100
@@ -97,6 +97,7 @@
 #endif
 }
 
+#if defined(LOADER_DYNAMIC_LIB)
 BOOL WINAPI DllMain(HINSTANCE hinst, DWORD reason, LPVOID reserved) {
     (void)hinst;
     switch (reason) {
@@ -114,6 +115,7 @@
     }
     return TRUE;
 }
+#endif
 
 bool windows_add_json_entry(const struct loader_instance *inst,
                             char **reg_data,    // list of JSON files
diff -ruN a/loader/vk_loader_platform.h b/loader/vk_loader_platform.h
--- a/loader/vk_loader_platform.h	2024-04-19 17:48:08.000000000 +0100
+++ b/loader/vk_loader_platform.h	2024-04-20 07:46:00.947325106 +0100
@@ -71,15 +71,12 @@
 #include <io.h>
 #include <shlwapi.h>
 #include <direct.h>
+#include <pthread.h> // for mingw
 #endif  // defined(_WIN32)
 
 #include "stack_allocation.h"
 
-#if defined(APPLE_STATIC_LOADER) && !defined(__APPLE__)
-#error "APPLE_STATIC_LOADER can only be defined on Apple platforms!"
-#endif
-
-#if defined(APPLE_STATIC_LOADER)
+#if defined(BUILD_STATIC_LOADER)
 #define LOADER_EXPORT
 #elif defined(__GNUC__) && __GNUC__ >= 4
 #define LOADER_EXPORT __attribute__((visibility("default")))
@@ -233,7 +230,7 @@
 // resources allocated by anything allocated by once init. This isn't a problem for static libraries, but it is for dynamic
 // ones. When building a DLL, we use DllMain() instead to allow properly cleaning up resources.
 
-#if defined(APPLE_STATIC_LOADER)
+#if defined(BUILD_STATIC_LOADER)
 static inline void loader_platform_thread_once_fn(pthread_once_t *ctl, void (*func)(void)) {
     assert(func != NULL);
     assert(ctl != NULL);
diff -ruN a/loader/vulkan_own.pc.in b/loader/vulkan_own.pc.in
--- a/loader/vulkan_own.pc.in	1970-01-01 01:00:00.000000000 +0100
+++ b/loader/vulkan_own.pc.in	2024-04-20 07:46:00.947325106 +0100
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
+
+Name: Vulkan-Loader
+Description: Vulkan Loader
+Version: @VULKAN_LOADER_VERSION@
+Libs: -L${libdir} -lvulkan
+Libs.private: -lshlwapi -lcfgmgr32
+Cflags: -I${includedir}
