diff -ruN a/libcc1/libcc1plugin.cc b/libcc1/libcc1plugin.cc
--- a/libcc1/libcc1plugin.cc	2023-04-26 14:36:39.566516284 +0100
+++ b/libcc1/libcc1plugin.cc	2023-04-26 14:38:14.616485534 +0100
@@ -33,6 +33,7 @@
 
 #define INCLUDE_MEMORY
 #include "gcc-plugin.h"
+#include "pthread.h"
 #include "system.h"
 #include "coretypes.h"
 #include "stringpool.h"
diff -ruN a/libcc1/libcp1plugin.cc b/libcc1/libcp1plugin.cc
--- a/libcc1/libcp1plugin.cc	2023-04-26 14:36:39.566516284 +0100
+++ b/libcc1/libcp1plugin.cc	2023-04-26 14:38:14.616485534 +0100
@@ -34,6 +34,7 @@
 
 #define INCLUDE_MEMORY
 #include "gcc-plugin.h"
+#include "pthread.h"
 #include "system.h"
 #include "coretypes.h"
 #include "stringpool.h"
